.golangci.yml

```diff
@@ -0,0 +1,5 @@
+linters:
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default
+  enable:
+    - staticcheck
```

Comment:
I think now that we have this, we could also enable `govet` here?

Reply:
Seems like by default there are also `errcheck, gosimple, ineffassign, unused` enabled. I kept them on.